### PR TITLE
post request to save new event input to db

### PR DESCRIPTION
**Pull request by htthuyen**

### CODE REVIEW

File: prisma/schema.prisma [hunk 1]
```diff
@@ -61,3 +61,12 @@ model Event {
   // As described in the User model, User and Event have a many to many relationship
   // so we only need a relationship attribute to disambiguate attendees and signedUpUsers
 }
+// this table will store new future events, input by TOF admin
+model eventCreation {
```
Comment (md-y) [inline prisma/schema.prisma]: Why can't we use the normal event model?

Comment (htthuyen) [reply prisma/schema.prisma]: I thought about it, too. But, I haven't figured out how to select some needed attributes for event listing page. So, I created a new model to see if my post request worked. I'll do some more research

Comment (md-y) [reply prisma/schema.prisma]: I was able to make an event using arbitrary strings / data like so:

```javascript
const eventListing = await prisma.event.create({
    data: {
      name: 'test event',
      organizer: 'texas osteo',
      location: 'Somewhere, Texas',
      dateAndTime: new Date(dateTime),
      duration: 1,
      thumbnail: 'test.png',
      hoursOffered: 3,
      phoneNumber: '(123) 456-7890',
      email: 'test@example.com',
      isSignUpAvailable: true,
      capacity: 100,
      attendees: {
        connect: {
          id: 'example id',
        },
      },
    },
  })
```

Also the attendees and signed up users can be references by their ids, so the JOI schema can accept a string array for them.

Comment (htthuyen) [reply prisma/schema.prisma]: Cool! Let's use yours

**Review by md-y — APPROVED** [review]
looks good

**Pull request merged**